### PR TITLE
Implemented common floating point math functions

### PR DESCRIPTION
**Pull request by ColonelJ**

### CODE REVIEW

File: common.ktn [hunk 1]
```diff
@@ -230,6 +230,48 @@ vocab kitten {
   intrinsic div_float64 (Float64, Float64 -> Float64)
   intrinsic mod_float64 (Float64, Float64 -> Float64)
 
+  // Floating-point functions.
+
+  intrinsic pi_float32 (-> Float32)
```
Comment (evincarofautumn) [inline common.ktn]: `pi` could be a literal instead of an intrinsic.

Comment (ColonelJ) [inline common.ktn]: No worries, I will fix that

**Comment by ColonelJ** [comment]
OK, sqrt and atan2 have been added, pi is now just implemented with the shortest possible literals, and the commit has been rebased against master to resolve conflicts, so everything should now be good to go!

**Comment by evincarofautumn** [comment]
Awesome, thank you!

**Pull request merged**